include/grpc/grpc_security.h

```diff
@@ -121,39 +123,6 @@ GRPCAPI void grpc_ssl_session_cache_destroy(grpc_ssl_session_cache* cache);
 GRPCAPI grpc_arg
 grpc_ssl_session_cache_create_channel_arg(grpc_ssl_session_cache* cache);
```

Comment:
The SSL session cache stuff is also used by SslCreds / TlsCreds, so it should move where they are.